src/components/ErrorModal.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="modal-mask"
+    @click="resolve"
+  >
+    <div class="modal-wrapper" @click.stop>
+      <div class="error-modal">
+        <h1>{{ $t('reportBug') }}</h1>
+        <textarea
+          v-model="report"
+          class="report-text"
+        />
+        <button
+          class="button"
+          @click="send"
+        >
+          {{ $t('sendBugReport') }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { detect } from 'detect-browser';
+
+import SuccessIcon from '../assets/verifiedUrl.svg';
+import FailureIcon from '../assets/iconError.svg';
+import { backendFetch } from '../utils/backend';
+
+export default {
+  props: {
+    failure: { type: Boolean },
+    hideIcon: { type: Boolean },
+    resolve: { type: Function, required: true },
+    error: { type: [String, Number], default: 'Something went wrong' },
```

Comment:
In which case, Error can be a number? If somebody does like `throw 42`?

```suggestion
    error: { type: Error, required: true },
```